lms/templates/header/navbar-authenticated.html

```diff
@@ -61,7 +61,7 @@
         </div>
       % endif
       <div class="mobile-nav-item hidden-mobile nav-item nav-tab">
-          <a class="${'active ' if '/u/' in request.path  else ''}tab-nav-link" href="${reverse('learner_profile', args=[self.real_user.username])}"
+          <a class="${'active ' if '/u/' in request.path  else ''}tab-nav-link" href="${urljoin(settings.PROFILE_MICROFRONTEND_URL, f'/u/{self.real_user.username}')}"
```

Comment:
This itself looks fine but I wanted to take a deeper look at some (Not so) recent changes in the platform and i noticed that there have been some recent changes in the platform that we might adopt in the theme that might keep us safe from a broken theme in the future as well. I noticed:

1. The profile link/tab has been removed from edX, [See reference](https://github.com/openedx/edx-platform/pull/19796/files#diff-eeb3ae2ade6d0b592654a00afea3a4e581b54d8e36ca67b57eb3fe55eb7ca754) and at this point, I think we should also remove this section from our theme as well instead of removing just the `reverse`. _Why I think this should be removed_ is because we usually redirect our users to the Marketing site(MITxOnline) for the `/dashboard` and don't really need this tab here as well. Here is how this tab looks: (Might be better to get @pdpinch 's opinion here)
<img width="714" alt="Programs copy" src="https://user-images.githubusercontent.com/34372316/223687413-fabcd01b-ee35-448d-a1fb-4056df3a6002.png">

2. I also noticed that the `Journals` section has also been removed from edX ([See Reference](https://github.com/openedx/edx-platform/commit/2583d317db73bd02fe6134eb0df89834defd0b15#diff-eeb3ae2ade6d0b592654a00afea3a4e581b54d8e36ca67b57eb3fe55eb7ca754)) But this might be something that we can/should do in another PR since it's breaking anything at this point. I'll create a ticket to remove these.

**Side Note**: Similar changes might be required in xPRO/MITx themes as well. I'll try to create tickets for all these.

FYI, @pdpinch

Reply:
This looks good.

> Thought: should we also pull the Profile link in the user dropdown? It's not breaking things but it seems inconsistent to me to pull one but not the other. (It's still there in edX main though.) This shouldn't block this PR, though.

The `Profile` option in the drop-down redirects the user to the marketing site, e.g. (`mitxonline/profile`). But in the case of the tab, it takes the user to edX's own profile page e.g. (`https://courses-qa.mitxonline.mit.edu/u/<username>`).